docs/agent-config.adoc

```diff
@@ -0,0 +1,54 @@
+= Agent configuration
+
+Agent can be run with a configuration by invoking `hercules-ci-agent --config FILE.toml`.
```

Comment:
```suggestion
The agent can be run with a configuration by invoking `hercules-ci-agent --config agent.toml`.
```

"The" + modify the filename to match the suggested filename at the example. No need to use a metavariable.